fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/api/attachment/v1/AttachmentRegistry.java

```diff
@@ -139,6 +143,41 @@ public interface Builder<A> {
 		 */
 		Builder<A> initializer(Supplier<A> initializer);
 
+		/**
+		 * Indicates that the attached data should be synced with all possible clients.
+		 *
+		 * @return the builder
+		 */
+		Builder<A> syncWithAll(PacketCodec<PacketByteBuf, A> packetCodec);
```

Comment:
It would have to be some sort of wrapper around a BiPredicate, the point of these specialized methods in the impl is that they determine an enum flag that allows for simplification down the line. Certainly doable though, would be nice for uniformization